cmd/service/dev.go

```diff
@@ -95,9 +97,21 @@ func listenResults(serviceID string, result string, output string) {
 	for {
 		result, err := stream.Recv()
 		if err != nil {
-			logrus.Info(aurora.Red(err))
+			log.Println(aurora.Red(err))
 			return
 		}
-		logrus.Info("Receive result", aurora.Green(result.TaskKey), aurora.Cyan(result.OutputKey), "with data", aurora.Bold(result.OutputData))
+		log.Println("Receive result", aurora.Green(result.TaskKey), aurora.Cyan(result.OutputKey), "with data", formatJSON(result.OutputData))
+	}
+}
+
+func formatJSON(data string) string {
```

Comment:
Move to utils?

Missing docs.

s/formatJSON/colorJSON/ ?

Reply:
In which utils your suggest to move it? `service/utils`? `cmd/utils`?

Reply:
cmd/utils